packages/fury-adapter-remote/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "fury-adapter-remote",
+  "version": "0.0.1",
+  "description": "Provide adapter for Element API based on callin remote API site",
+  "author": "Apiary.io <support@apiary.io>",
+  "license": "MIT",
+  "main": "./lib/adapter.js",
+  "homepage": "https://github.com/apiaryio/api-elements.js/tree/master/packages/fury-adapter-remote",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apiaryio/api-elements.js.git"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "mocha test"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "dependencies": {
+    "axios": "^0.18.0",
+    "deckardcain": "^0.4.1",
+    "fury": "^3.0.0-beta.8"
```

Comment:
This should become a peerDependency and a devDependency, check how they are defined in other adapters (`fury-adapter-oas3-parser/package.json`).

Since a user of the adapter is expected to depend on Fury, having a hard dependency will cause a duplicate dependency.

Reply:
ok